src/main/java/org/fogbowcloud/arrebol/ArrebolController.java

```diff
@@ -62,54 +81,61 @@ public ArrebolController(Properties properties) {
         this.queue = new TaskQueue(queueId, queueName);
 
         int poolId = 1;
-        WorkerPool pool = createPool(arrebolProperties, poolId);
+        WorkerPool pool = createPool(configuration, poolId);
 
         //create the scheduler bind the pieces together
         FifoSchedulerPolicy policy = new FifoSchedulerPolicy();
         this.scheduler = new DefaultScheduler(queue, pool, policy);
 
-        this.jobPool = Collections.synchronizedMap(new HashMap<String,  Job>());
+        this.jobPool = Collections.synchronizedMap(new HashMap<String, Job>());
         this.jobDatabaseCommitter = new Timer(true);
+        this.jobStateMonitor = new Timer(true);
     }
 
-    private static final String RAW_TYPE = "raw";
-    private static final String DOCKER_TYPE = "docker";
+    private void certifyConfigurationProperties() throws Exception {
```

Comment:
why are these local variables final?

Reply:
Because the name of each type stays the same. But these variables were removed from this location.